### PR TITLE
[AzureLogAnalytics] update docs

### PR DESCRIPTION
**Pull request by michal-dagan**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
relates: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-33821)

## Description
Documentation and metadata improvements.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AzureLogAnalytics/ReleaseNotes/1_1_27.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+
+##### Azure Log Analytics
+- Documentation and metadata improvements.
```
Comment (michal-dagan) [inline Packs/AzureLogAnalytics/ReleaseNotes/1_1_27.md:5]:
```suggestion
Documentation and metadata improvements.
```

File: Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics_description.md [hunk 2]
```diff
@@ -2,18 +2,19 @@ Log Analytics is a service that helps you collect and analyze data generated by
...
 
-1. Access the [authorization flow](https://oproxy.demisto.ninja/ms-azure-log-analytics). 
-2. Click the **Start Authorization Process** button. 
-   You will be prompted to grant Cortex XSOAR permissions for your Azure Service Management. 
-3. Click the **Accept** button. 
-   You will receive your ID, token, and key. You need to enter this information, when you configure the Azure Log Analytics integration instance in Cortex XSOAR.
+#### Cortex XSOAR Azure App
 
-## Authorize Cortex XSOAR for Azure Log Analytics - Self-Deployed Configuration
+To use the **Cortex XSOAR application** and allow Cortex XSOAR access to Azure Log Analytics, an administrator has to approve our app using an admin consent flow by clicking this **[link](https://oproxy.demisto.ninja/ms-azure-log-analytics)**.
```
Comment (ShirleyDenkberg) [inline Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics_description.md:14]: Not sure what you mean by "our app".  Please change the wording so there is no doubt what app you are referring to.

File: Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/README.md [hunk 3]
```diff
@@ -1,29 +1,33 @@
...
 
-In addition, the user needs to be assigned the **Log Analytics Reader** role.
-To add the role, see the [Microsoft article](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-add-app-roles-in-azure-ad-apps#assign-users-and-groups-to-roles).
+In addition, the user that granted the authorization needs to be assigned the [**Log Analytics Reader** role](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/manage-access?tabs=portal#log-analytics-reader). 
+For the search job commands the user needs to be assigned the [**Log Analytics Contributor** role](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/manage-access?tabs=portal#log-analytics-contributor).
+
+    To add these roles:
+     - In the Azure portal, go to `Log Analytics workspace` and select the workspace you are using -> Access control (IAM).
+     - From Access control (IAM) select: Add role assignment
+     - Select the user that granted the authorization and assign the Roles.
```
Comment (ShirleyDenkberg) [inline Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/README.md:21]:
```suggestion

      1. In the Azure portal, go to `Log Analytics workspace` and select the workspace you are using -> Access control (IAM).
      2. From Access control (IAM) select: Add role assignment
      3. Select the user that granted the authorization and assign the Roles.

```

**Comment by ShirleyDenkberg** [comment]
@michal-dagan Doc review completed.

File: Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics_description.md (hunk 2 above)
Comment (michal-dagan) [reply Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics_description.md:14]: like here: https://github.com/demisto/content/pull/31866#discussion_r1440524833

**Review by bziser — APPROVED** [review]

**Pull request merged**